<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColourSelectionWindow</class>
 <widget class="QDialog" name="ColourSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Color</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>20</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="RedHorizontalSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="RedSpinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="GreenHorizontalSlider_3">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="GreenSpinBox_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="BlueHorizontalSlider_2">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="BlueSpinBox_3"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="NormalMapCheckBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use normal map</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>RedHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RedSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GreenHorizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GreenSpinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BlueHorizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BlueSpinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RedSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RedHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GreenSpinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GreenHorizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BlueSpinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BlueHorizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
